<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reports</class>
 <widget class="QDialog" name="Reports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #55ffff</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>261</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Enter Report Information:</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_date">
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_date">
        <property name="styleSheet">
         <string notr="true">background-color: #ffffff</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_membership">
        <property name="text">
         <string>Membership Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_membership">
        <property name="styleSheet">
         <string notr="true">background-color: #feff57</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">background-color: #feff57</string>
      </property>
      <property name="text">
       <string>Generate Report</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_tableTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Generated Report:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_miniLogo">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_totalRevTitle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Revenue:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_totalRev">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffffff</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_titleItemsSold">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Items Sold:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_titleShoppers">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Shoppers:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>89</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_titleRegMembers">
      <property name="text">
       <string>Regular Members:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_regMember">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>99</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_titleExecMember">
      <property name="text">
       <string>Executive Members:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_execMember">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget_shoppers">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>211</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffffff</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffffff</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
